src/helm/proxy/retry.py

```diff
@@ -52,5 +52,5 @@ def retry_if_request_failed(result: Union[RequestResult, TokenizationRequestResu
 
 
 retry_request: Callable = get_retry_decorator(
-    "Request", max_attempts=8, wait_exponential_multiplier_seconds=5, retry_on_result=retry_if_request_failed
+    "Request", max_attempts=1, wait_exponential_multiplier_seconds=5, retry_on_result=retry_if_request_failed
```

Comment:
Revert this (though I generally agree that max_attempts should be <= 5).

@teetone are you still relying on high retry count?

Reply:
Is there a flag for this? I feel like we are all changing this locally, maybe it's time to add a flag?

Reply:
my mistake thanks!